src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue24856.cs

```diff
@@ -19,7 +19,9 @@ public void ImageButtonAspectFitWorks()
 		{
 			App.WaitForElement("WaitForStubControl");
 			App.Tap("UpdateAspect");
-			VerifyScreenshot();
+
+			Thread.Sleep(1000);
+			VerifyScreenshot(retryDelay: TimeSpan.FromSeconds(2));
```

Comment:
I think we just build this retry into VertifyScreenshot by default for Android so we don't have to pass it

Reply:
Yeah, just increase the waiting time between snapshots.